brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1705 Fix: Post Layout: Title content not visible after title html tag P is set.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1705

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Post grid, masonry, carousel.
- Set Title HTML Tag to P tag.
- See if it is visible or not.
- Try adding color, spacing, and typography to the title. Everything should work fine

### Checklist:
